android/app/src/main/java/com/rohtvapp/ROHBitMovinPlayerPackage/PlayerContainerView.java

```diff
@@ -90,13 +91,15 @@ public void init() {
         audioCodecPriority.add("mp4a.a5");
         audioCodecPriority.add("mp4a.40");
 
+        boolean shouldEnableTunneledPlayback = Build.MANUFACTURER.contains("Google") || Build.MANUFACTURER.contains("Amazon");
```

Comment:
Will this fail gracefully on tvOS?

Reply:
Since tvOS's manufacturer is Apple, it won't even reach this point. The Java modules are built specifically for Android.

We'll have to implement similar logic for tvOS in Objective-C when we implement the Bitmovin module for it.

Even if it did reach that point on tvOS, the expression would evaluate to `false` - there are no risks of exceptions being thrown.

Reply:
> it won't even reach this point. The Java modules are built specifically for Android.

of course, I'm an idiot, sorry :)